### PR TITLE
Remove mail catcher

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
Remove mail catcher as we have moved from Spring Email to GOV.UK Notify.

### CODE REVIEW

File: src/main/resources/application-dev.yml [hunk 1]
```diff
@@ -45,4 +43,4 @@ app:
     region: eu-west-2
     bucket-name: cjsm-orgs
   notify:
-    apikey: dummy-api-key
```
Comment (adaviesMOJ) [inline src/main/resources/application-dev.yml:48]: is this change intended?

Comment (dhirajshettymoj) [reply src/main/resources/application-dev.yml:48]: thanks good spot. Needs to be dummy-api-key

File: src/testIntegration/resources/application-test.yml [hunk 2]
```diff
@@ -29,10 +29,6 @@ spring:
...
...bXxqR5LAVhht6G+Eckal/UP6fvh6QNuupiZJZ3QNrVPDVFsI16do+0mDLrHKNIy//+Tdnw8LMiBEmfb0Jetk+on5jeVCdPx+MK7kUCX5MvhNcIVYV6uD2XTUtXYxkveCAPOHuOlI9LXrj7UKofQyS1KalmNAoHBAMSVIAXsRQGBACSwhMiL84chjW3/GFyOl7gWnu2b/U8miCVoLaDGj4JZS4q8qEAzs/K2pTr4TzvNCB8iB2WUAehz81U3KDtUErSKJoqw33n9uEQW35kVjjbXTEZVWd8Wa7mavq6ZM/kALSC9Hvp4jKDHND8na+VuNHrFSgXNj3IAbDv+rcXXrgF8G6NxfYoZ1qt1f+HaC6ElaaEAV9hm9wDHsmvbg1p/AVw6Yt2cRG1UmtZySK5plqHsgsAOA8005wKBwCGkvIal4O6JcCM4NhuF9EncqMpdq/qxt/Q4z1E20uUsPi92whSfWqskij5mJ/NcD2mL3PBUWrXYL21Pwuvyk5s465NLDIVL5LbgzsjR8KCrVkNKiBL4oZfdSn109MLpvz/mSQSM1fTWKdhb3kWb//F1URoPqg6/XsjkoXSGeZM4udPI3UBj0GR+Hyodm6bj07hjM6yKsNQriuE4nlN3Rn9gVeRELq2ihrVtHZjEx+FgOYPG6brasbKx40TKK1OOoQKBwHphF9l4GBFynIM6vL8YmRez5JboquLcBkmdE316kbyXkUQwUc94LVkJfAS3o8qsJUdhOwW2FaV146t2BMLQ3Es4+/A98TbWBPsN0iafK9CHChKhZrundqxfZeWXNuaxz4hxdGm/iGJoIK/z7HgjWfCxcq4qPXpMHDGd5pIsQTPIXZkP2GCE/5d/Crm6alvx7/k3odAZbQn5/4Mycn2OxBBhQYrx0hkhXdhrpKavpM2nUOlVpjemFhdh5JfpYaZ7WwKBwBpClMB7+ObSwpiS0sC0qLrQ1wbtyTOESypp9zM9Tto4dNnR1LA6T/p1AGt7pyFHK40j5ekTnTqUKsV4Qh5cFjeQpLyoYxqBfYncfbnevW/fFQU0aP6lOktUo7nLO6WXYyx+MsXB+fnPtk08XkR/c/uom913xYXihvERLReIGypxkTFG1iu1XlJw4sGyeeRaU//bawGFlzN06H2kzNMrZ5LwHmVmsoUdnmMtwNlLBCG9iGsWt56672nuv2vfPd6xEg=='
```
Comment (adaviesMOJ) [inline src/testIntegration/resources/application-test.yml:34]: not apart of the PR, but is this supposed to be here?

Comment (dhirajshettymoj) [reply src/testIntegration/resources/application-test.yml:34]: not sure. will look into it as part of  a different PR

**Review by adaviesMOJ — APPROVED** [review]
